etc/init/spinnaker.conf

```diff
@@ -31,3 +45,7 @@ pre-start script
     start $i
   done
 end script
+
+respawn
```

Comment:
I don't think the top-level spinnaker upstart entry will be aware of the subservice health. This needs to go in orca's upstart.